### PR TITLE
Fix/499 bop summary

### LINKED ISSUE #499: Add Summary Table to BOP Page
## Objective
Add summary table (same designs as state summary tables) to BOP page. 

Yes, that summary table looks great! I think as long as it's clear that it's about Bureau of Prisons, which the page header would make clear I believe.

## Workflow Checklist
- [x] [UCLA] Provide feature requirements
- [x] [HO] Draft feature specification
- [x] [UCLA] Approve feature specification
- [x] [HO] Mockup designs and plan of approach
- [x] [UCLA] Feedback round (feedback on how closely designs match agreed upon feature spec) 
- [x] [HO] Implement design feedback
- [x] [UCLA] Approve designs
- [ ] [HO] Dev implementation demo (iterative) 
- [ ] [UCLA] Provide feedback (on ways in which dev implementation differs from designs and agreed upon functionality) 
- [ ] [HO] Final dev implementation demo
- [ ] [UCLA] Approve feature implementation for prod release

### PR DESCRIPTION
**Pull request by hoshmn**
todo:
- ~~CMSify~~

fixes #499

### CODE REVIEW

File: content/lang/federal.json [hunk 1]
```diff
@@ -1,30 +1,13 @@
...
-    "unavailable": "${metric} is not available for staff.",
-    "notes": {
-      "active": "True case counts are likely higher and may be significantly higher than reported",
-      "confirmed": "True case counts are likely higher and may be significantly higher than reported",
-      "deaths": "True case counts are likely higher and may be significantly higher than reported",
-      "tests": ""
-    }
+  "aggregates": {
+    "id": "aggregates",
+    "description": "The table below displays aggregate COVID-19 data for incarcerated people and staff in carceral facilities located in ${state}. The State DOC Data Grade represents our assessment of the transparency and quality of those data reported by ${state}'s state department of corrections.",
```
Comment (hoshmn) [inline content/lang/federal.json:4]: I realize some of this copy is state-specific, will fix when I migrate it to the CMS

File: src/templates/federal.js [hunk 2]
```diff
@@ -72,13 +73,24 @@ const useStyles = makeStyles((theme) => ({
       "& .step-wrapper": {
         width: "unset",
       },
-      maxWidth: "44rem",
+      maxWidth: "45rem",
+
+      // HACK: agg table overrides, need to restyle this page but this is here for now
+      "& #aggregates .MuiTableContainer-root": {
+        width: "calc(100vw - 6rem)",
```
Comment (hoshmn) [inline src/templates/federal.js:80]: @Lane see what you think of the styling, I figured we should let it expand to the full width available

**Comment by ekmason** [comment]
@Lane  @hoshmn Small note - do we want to keep the "Federal Prisons" header in the summary table (screenshot below)? 
image

(This header wasn't present on Lane's original design shared in the ticket)
image

**Comment by Lane** [comment]
@ekmason i think it's fine to keep the label on the table, it will provide some additional context if someone were to screenshot the table only.

@hoshmn looking good so far, a few changes:

- can we make the layout consistent with the state pages?  specifically:
  - spacing between header and top navigation
  - spacing between header text and intro paragraph
  - overall width of the content area (e.g. table spans full width of content area)
  - placement of the data grade

ideally these would match up:


also checking in, are the CMS pieces ready to test?

**Comment by hoshmn** [comment]
thanks both for the feedback. @Lane i added styling fixes to match the states page.
>       * overall width of the content area (e.g. table spans full width of content area)
the federal page has a very restricted content area (45rem), hence all the css workarounds for elements that need to exceed that width. i currently have styles on the table to have it break out of the content area and grow to its full width when there's space.
> also checking in, are the CMS pieces ready to test?
yes! sorry, forgot to ping you when i added that

**Comment by Lane** [comment]
thanks @hoshmn, i'm still seeing a layout shift in table width / placement of the section between the federal and state pages.

ideally, the federal + state templates should use similar content widths.  i've put in a refactor PR (#563) for the federal template that cleans up the layout and drops code that's no longer needed.  can you review and merge into this branch if it looks good to you?

once that's done, i'll merge this to development (along with other PR reviews) and do a similar refactor on the state template to remove the pieces we're no longer using

**Comment by hoshmn** [comment]
Thanks for the refactor @Lane, looks great! Just merged it in.

> thanks @hoshmn, i'm still seeing a layout shift in table width / placement of the section between the federal and state pages.

The shift you're seeing should be addressed by the state template refactor, right? If not I'll fix.

**Comment by Lane** [comment]
@hoshmn yep, the state refactor will bring the two layouts inline with each other.  thanks!

**Pull request merged**